subsys/mpsl/pm/Kconfig

```
@@ -15,6 +15,19 @@ config MPSL_USE_ZEPHYR_PM
 
 if MPSL_USE_ZEPHYR_PM
 
+config MPSL_PM_UNINIT_WORK_WAIT_TIME_MS
+	int "Time the MPSL PM uninit procedure will wait for work queue to complete in milliseconds"
```

I would slightly rephrase it:

```suggestion
	int "Time in milliseconds that the MPSL PM uninit procedure will wait for the work queue to complete"
```

---

Maybe: `"Timeout value the MPSL PM uninit procedure will wait for work queue to complete, in milliseconds"`?